packages/block-editor/src/components/block-list/style.scss

```diff
@@ -319,6 +319,11 @@
 	}
 }
 
+.is-root-container > .alignfull {
+	margin-right: calc(var(--wp--style--padding-right) * -1);
+	margin-left: calc(var(--wp--style--padding-left) * -1);
```

Comment:
Not sure if we should have a fallback value here in case the variable doesn't exist.